src/Microsoft.Data.Analysis/DataFrameColumnCollection.cs

```diff
@@ -141,5 +141,26 @@ protected override void ClearItems()
             _columnNames.Clear();
             _columnNameToIndexDictionary.Clear();
         }
+
+        /// <summary>
+        /// Searches for a <see cref="DataFrameColumn"/> with the specified <paramref name="name"/> and attempts to return it as <typeparamref name="T"/>. If <typeparamref name="T"/> does not match <see cref="DataFrameColumn.DataType"/>, returns null.
+        /// </summary>
+        /// <typeparam name="T">The type to cast the column to</typeparam>
+        /// <param name="name">The name of the column</param>
+        /// <returns><see cref="DataFrameColumn"/> as <typeparamref name="T"/>. Null if types don't match</returns>
```

Comment:
Your doc and your behavior doesn't seem to match. The doc says it returns null if the types don't match. But the code throws an ArgumentException.

Reply:
Word wrap issues :/. Fixed